dist/ce-bind.js

```diff
@@ -180,11 +180,40 @@ function ceOneWay() {
     scope: false,
     compile: function($element, $attrs) {
 
+      // parse an expression and returns its metadata
+      function parse(expression) {
+
+        // keep a local cache so we don't parse the same expression again and again...
+        var cache = {};
+
+        return function() {
+          if (!cache[expression]) {
+
+            // cache[expression][0] = expression
+            // cache[expression][1] = controller alias (with dot) i.e. "$ctrl." or "undefined" for non-aliased controllers
+            // cache[expression][2] = controller alias (without dot) i.e. "$ctrl" or "undefined" for non-aliased controllers
+            // cache[expression][3] = event handler
+            // cache[expression][4] = event handler params (i.e. "$event, a, b, c")
+
+            cache[expression] = expression.match(/((.*)\.)?(\w*)\((.*)\)/);
+          }
+          return cache[expression];
```

Comment:
What do you think about returning an object that has properties for the controller name and the handler. That way lines 208 and 216 could depend on those properties instead of numbered keys of the array